backend/apps/search/management/commands/reindex_meilisearch.py

```
@@ -0,0 +1,57 @@
+                "description": doc.description,
+                "tags": doc.tags,
+                "body_text": doc.body_text,
+                "content_type_name": doc.content_type_name,
+            })
+
+            if len(batch) >= batch_size:
+                try:
+                    index.add_documents(batch)
+                    indexed_count += len(batch)
+                    self.stdout.write(f"Indexed {indexed_count}/{total_docs} documents...")
+                except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)